doc/services/storage/zms/zms.rst

```diff
@@ -414,6 +415,41 @@ verified to make sure that the application could work with one subsystem or the
 both solutions could be implemented, the best choice should be based on the calculations of the
 life expectancy of the device described in this section: :ref:`wear-leveling`.
 
+Recommendations to increase performance
+***************************************
+
+Sector size and count
+=====================
+
+- The total size of the storage partition should be well dimensioned to achieve the best
```

Comment:
As I've already commented elsewhere, I don't think `dimensioned` is the best choice here. e.g. simpler alternative here (either-or):
```suggestion
- The total size of the storage partition should be set carefully/appropriately to achieve the best
```